themes/bootstrap5/templates/admin/config/home.phtml

```diff
@@ -12,5 +12,5 @@
 </div>
 
 <div class="<?=$this->layoutClass('sidebar')?>">
-  <?=$this->adminMenu()->render()?>
+  <?=$this->section('AdminMenu', \VuFind\Section\SectionServiceInterface::DEFAULT_CONFIG_FILE, 'admin/menu.phtml')->render()?>
```

Comment:
I would suggest using named arguments in all of these templates so you don't have to specify that long constant name when it's the default anyway:


```suggestion
  <?=$this->section('AdminMenu', template: 'admin/menu.phtml')->render()?>
```